index.ts

```diff
@@ -146,8 +183,17 @@ async function analyzeProject(project: Project): Promise<ProjectResult> {
         args.push(project.typesPath);
     }
 
+    // If it's going to include a JSON path, make it per-trace
+    let jsonPath: string | undefined = undefined
+    if (opts.includes("--json")) {
+        const hash = crypto.createHash('sha256').update(project.tracePath).digest("hex");
```

Comment:
Why are we hashing files?  I don't think there's a way to update one trace out of a family - are we guarding against having to redo work on crash?